--- a/MODULE.bazel	2026-08-29 20:09:21.776525358 +0000
+++ b/MODULE.bazel	2026-08-29 20:09:21.788819446 +0000
@@ -17,15 +17,13 @@
 bazel_lib_toolchains.coreutils()
 bazel_lib_toolchains.zstd()
 bazel_lib_toolchains.expand_template()
-bazel_lib_toolchains.bats()
-use_repo(bazel_lib_toolchains, "bats_toolchains", "copy_directory_toolchains", "copy_to_directory_toolchains", "coreutils_toolchains", "expand_template_toolchains", "zstd_toolchains")
+use_repo(bazel_lib_toolchains, "copy_directory_toolchains", "copy_to_directory_toolchains", "coreutils_toolchains", "expand_template_toolchains", "zstd_toolchains")
 
 register_toolchains(
     "@copy_directory_toolchains//:all",
     "@copy_to_directory_toolchains//:all",
     "@coreutils_toolchains//:all",
     "@expand_template_toolchains//:all",
-    "@bats_toolchains//:all",
     "@zstd_toolchains//:all",
 )
 
